staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/errors.go

```diff
@@ -14,9 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package webhook
+package config
 
-// AdmissionConfig holds config data that is unique to each API server.
-type AdmissionConfig struct {
-	KubeConfigFile string `json:"kubeConfigFile"`
+import "fmt"
+
+type ErrCallingWebhook struct {
```

Comment:
// ErrCallingWebhook is returned for transport-layer errors calling webhooks. It represents a failure to talk to the webhook, not the webhook rejecting a request.

Or something.

Reply:
Done.